app/src/main/java/com/example/desafios_zup/view/search/SearchFragment.kt

```diff
@@ -0,0 +1,88 @@
+package com.example.desafios_zup.view.search
+
+import android.graphics.Color
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.SearchView
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.example.desafios_zup.R
+import com.example.desafios_zup.viewModel.SearchViewModel
+
+class SearchFragment : Fragment() {
+    private lateinit var searchViewModel: SearchViewModel
+    val mAdapter: RecyclerViewSearchAdapter = RecyclerViewSearchAdapter()
+    lateinit var recyclerViewMovies: RecyclerView
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        searchViewModel = ViewModelProvider(this).get(SearchViewModel::class.java)
+
+        var rootView = inflater.inflate(R.layout.fragment_search, container, false)
+
+        val searchIcon = rootView.findViewById(R.id.search_mag_icon) as ImageView
+        searchIcon.setColorFilter(Color.WHITE)
+
+
+        val cancelIcon = rootView.findViewById(R.id.search_close_btn) as ImageView
+        cancelIcon.setColorFilter(Color.WHITE)
+
+
+        val textView = rootView.findViewById(R.id.search_src_text) as TextView
+        textView.setTextColor(Color.WHITE)
+
+
+        recyclerViewMovies = rootView.findViewById(R.id.recycler_view_movie) as RecyclerView
```

Comment:
eu substitui essa linha `recyclerViewMovies = rootView.findViewById(R.id.recycler_view_movie) as RecyclerView` por essa `recyclerViewMovies = recycle_view_movie` mas deu que o recyclerViewMovies era null. 
Esse foi o erro q apareceu:
`E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.example.treinobottomnavigationview, PID: 10576
    java.lang.IllegalStateException: recycle_view_movie must not be null
        at com.example.desafios_zup.view.search.SearchFragment.onCreateView(SearchFragment.kt:45)`